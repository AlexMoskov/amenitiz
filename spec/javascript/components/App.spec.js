import React from 'react'
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from 'components/App'

configure({ adapter: new Adapter() });

describe('App', () => {
    const props = {}
    const component = shallow(<App {...props} />)

    it('renders App component', () => {
        expect(toJson(component)).toMatchSnapshot()
    })
})
