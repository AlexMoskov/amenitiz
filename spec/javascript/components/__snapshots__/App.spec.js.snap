// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App component 1`] = `
<Fragment>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/products"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/products/:id"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/cart"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/purchases"
      />
    </Switch>
  </BrowserRouter>
</Fragment>
`;
